app/jobs/descriptive_metadata_import_job.rb

```diff
@@ -44,11 +53,12 @@ def open_version(cocina_object, description)
     Failure([e.message])
   end
 
-  def validate_and_save(cocina_object, description)
-    result = CocinaValidator.validate_and_save(cocina_object, description:)
-    return Success(cocina_object) if result.success?
```

Comment:
note: the `CocinaValidator.validate_and_save` is used elsewhere in the code, so we will leave it as is even though we don't need it anymore in this class